docs/plugins/verifier/notation.md

```diff
@@ -155,6 +155,11 @@ Sample Notation CLI config:
                 }
             }
         ]
+    },
+    "crl": {
+      "cache": {
+        "enabled": false
```

Comment:
Is this the default value? If yes, it should be `true` as we discussed in last community meeting?

Reply:
Updated